control_panel_api/models.py

```diff
@@ -22,15 +26,33 @@ def get_short_name(self):
 
 
 class App(TimeStampedModel):
+    def _slugify(name):
```

Comment:
Should this be an instance method? e.g. pass `self` and replace name with `self.name`:

```python
    def _slugify(self):
        return re.sub(r'_+', '-', slugify(self.name))
```

Reply:
Oh hang on...it's because the signature of this function needs to be like that to work with `slugify_function`, sorry, didn't realise that.

Mmmm...wonder if we should move it up in the module (don't know if it's reusable in other models), but don't worry about that 👍

Reply:
@andyhd I've added a new ticket for this https://trello.com/c/vV7GOAPw/414-fix-slugify-function